src/assets/scss/components/playground-configuration.scss

```diff
@@ -152,13 +152,13 @@
 }
 
 .share-url {
-    display: inline-flex;
+    display: flex;
     align-items: center;
     gap: 1rem;
     background-color: var(--lightest-background-color);
     border-radius: var(--border-radius);
     border: 1px solid var(--border-color);
-    padding-right: calc(.625rem + 1vw);
+    padding-right: calc(.625rem + 20px);
```

Comment:
The share button is 20px width.

Reply:
`px` is an absolute length unit. If it is used, then `calc()` makes no sense.

Ref https://developer.mozilla.org/en-US/docs/Web/CSS/calc#accessibility_concerns

---

Also, can you explain why the `padding` needs to be modified? To solve the problem that share input does not have full width, it seems that you only need to change `inline-flex` to `flex`